### PR TITLE
Implement Cosmic Helm repo

### PR DESCRIPTION
**Pull request by bheuvel**
The general README has been adjusted to explain the usage of this (Helm) repo.
The `cosmic-microservices-chart` chart has been renamed to `cosmic-usage-chart`.
Grafana has been added to the _usage_ collection, and was partly the reason for implementing this repo.

Result is that implementation/usage of the `cosmic-usage-chart`, does not require a checkout of the charts source folder. Just add this (link to this) repo and use `helm install`.

After using `make` to package all charts, the contents of the `/repo` folder need to be uploaded to AWS S3, (from) where the actual Helm repository is hosted.

Grafana is copied from `stable/` pending a PR (https://github.com/kubernetes/charts/pull/941).
Once that is approved, `stable/grafana` can be used instead of this custom fork.

### CODE REVIEW

File: cosmic/microservices/cosmic-common/values.yaml [hunk 1]
```diff
@@ -2,3 +2,6 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
+# Following values are required to have sane values
+global:
+  namespace:
```
Comment (borisroman) [inline cosmic/microservices/cosmic-common/values.yaml]: \n

File: cosmic/microservices/cosmic-config-server/values.yaml [hunk 2]
```diff
@@ -8,3 +8,10 @@ image:
   pullPolicy: IfNotPresent
 service:
 
+# Following values are required to have sane values
+global:
+  vault:
+    scheme: https
+    host:
+    port:
+    backend:
```
Comment (borisroman) [inline cosmic/microservices/cosmic-config-server/values.yaml]: \n

File: cosmic/microservices/cosmic-metrics-collector/values.yaml [hunk 3]
```diff
@@ -7,3 +7,8 @@ image:
   tag: latest
   pullPolicy: IfNotPresent
 service:
+
+# Following values are required to have sane values
+global:
+  vault:
+    readOnlyToken:
```
Comment (borisroman) [inline cosmic/microservices/cosmic-metrics-collector/values.yaml]: \n

File: cosmic/microservices/cosmic-usage-api/values.yaml [hunk 4]
```diff
@@ -12,3 +12,9 @@ service:
     - port: 8080
       protocol: TCP
       nodePort: 31001
+
+# Following values are required to have sane values
+global:
+  namespace:
+  vault:
+    readOnlyToken:
```
Comment (borisroman) [inline cosmic/microservices/cosmic-usage-api/values.yaml]: \n

File: custom/cosmic-vault/values.yaml [hunk 5]
```diff
@@ -14,3 +14,7 @@ service:
       nodePort: 31101
 cosmicScanInterval: "0 */15 * * * *"
 rootToken:
+
+# Following values are required to have sane values
+global:
+  namespace:
```
Comment (borisroman) [inline custom/cosmic-vault/values.yaml]: \n

File: custom/rabbitmq/values.yaml [hunk 6]
```diff
@@ -10,3 +10,7 @@ service:
 username:
 password:
 
+# Following values are required to have sane values
+global:
+  namespace:
+
```
Comment (borisroman) [inline custom/rabbitmq/values.yaml]: \n

**Review by borisroman — CHANGES_REQUESTED** [review]
Store artifacts on S3

File: cosmic/microservices/cosmic-common/values.yaml (hunk 1 above)
Comment (bheuvel) [reply cosmic/microservices/cosmic-common/values.yaml]: Done

File: cosmic/microservices/cosmic-config-server/values.yaml (hunk 2 above)
Comment (bheuvel) [reply cosmic/microservices/cosmic-config-server/values.yaml]: Done

File: cosmic/microservices/cosmic-metrics-collector/values.yaml (hunk 3 above)
Comment (bheuvel) [reply cosmic/microservices/cosmic-metrics-collector/values.yaml]: Done

File: cosmic/microservices/cosmic-usage-api/values.yaml (hunk 4 above)
Comment (bheuvel) [reply cosmic/microservices/cosmic-usage-api/values.yaml]: Done

File: custom/cosmic-vault/values.yaml (hunk 5 above)
Comment (bheuvel) [reply custom/cosmic-vault/values.yaml]: Done

**Comment by bheuvel** [comment]
go build

**Comment by bheuvel** [comment]
go build

**Comment by bheuvel** [comment]
go build

**Comment by bheuvel** [comment]
go build

**Comment by bheuvel** [comment]
go build

**Pull request merged**